java/mtsj/core/src/main/java/com/devonfw/application/mtsj/ordermanagement/dataaccess/api/OrderedDishesPerDayEntity.java

```diff
@@ -0,0 +1,103 @@
+package com.devonfw.application.mtsj.ordermanagement.dataaccess.api;
+
+import java.sql.Timestamp;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.Immutable;
+
+import com.devonfw.application.mtsj.dishmanagement.dataaccess.api.DishEntity;
+import com.devonfw.application.mtsj.general.dataaccess.api.ApplicationPersistenceEntity;
+import com.devonfw.application.mtsj.ordermanagement.common.api.OrderedDishes;
+
+/**
+ * {@link ApplicationPersistenceEntity Entity} that represents a single
+ * {@link OrderedDishesPerDay} of an {@link OrderedDishesPerDay}.
+ */
+@Entity
+@Immutable
+@Table(name = "OrderedDishesPerDay")
+public class OrderedDishesPerDayEntity extends ApplicationPersistenceEntity implements OrderedDishes {
+
+  private static final long serialVersionUID = 1L;
+
+  private DishEntity dish;
+
+  // the number of ordered dished per day
```

Comment:
this is documentation that should go to the JavaDoc of the properties getter.

Reply:
done